project/config/independentpaneltruthrecoveryni/config/search_api.index.default_content.yml

```diff
@@ -133,6 +133,7 @@ field_settings:
 
 
 
+
```

Comment:
Not sure about this, we have had some problems with Solr on this site so we'll have to keep an eye on the edge site tomorrow to see if it breaks anything